config/initializers/govuk_navigation_helpers.rb

```diff
@@ -1,4 +1,3 @@
 GovukNavigationHelpers.configure do |config|
-  config.error_handler = Airbrake
-  config.statsd = Services.statsd
+  config.error_handler = GovukError
```

Comment:
You'll need `config.statsd = GovukStatsd` here like:

https://github.com/alphagov/collections/blob/master/config/initializers/govuk_navigation_helpers.rb#L1-L4

Reply:
👍